components/ProductMatrix.vue

```diff
@@ -40,17 +45,52 @@ export default Vue.extend({
       }
     }
   },
-  computed: mapGetters({
-    cards: 'cards/getCards',
-    loading: 'products/isLoading'
-  }),
+  computed: {
+    columns(): number {
+      const columns = Math.floor(this.windowWidth / 152)
+      return Math.max(2, columns)
+    },
+    cards(): Cards {
+      return this.allCards.cards
+    },
+    fillers(): Cards {
+      return this.allCards.filler.slice(0, this.fillersNeeded)
+    },
+    rows(): number {
+      return this.columns > 3 ? 3 : 999
```

Comment:
This might need a description or something. I guess calling it rowLimit and setting it to undefined or something would be easier to understand. Or use`Infinity`? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Infinity

Reply:
@coopjakob why are we using 999 for rows?

Reply:
Because mobile should show all rows, without any limit.

Reply:
Thanks! I'll try to make that clear